core/src/main/scala/za/co/absa/spline/harvester/plugin/embedded/DeltaPlugin.scala

```diff
@@ -76,8 +77,13 @@ class DeltaPlugin(
   }
 
   private def extractPath(command: AnyRef, fieldName: String): String = {
-    val targetFileIndex = extractValue[AnyRef](command, fieldName)
-    val path = extractValue[org.apache.hadoop.fs.Path](targetFileIndex, "path")
+    val path = Try {
+      val targetFileIndex = extractValue[AnyRef](command, fieldName)
+      extractValue[org.apache.hadoop.fs.Path](targetFileIndex, "path")
+    }.getOrElse {
```

Comment:
Mixed infix and prefix notation style. Either use `Try { ... } getOrElse { ... }` or `Try(...).getOrElse(...)`